src/hooks/useTheme.ts

```diff
@@ -0,0 +1,10 @@
+import { Theme } from 'gatsby-plugin-theme-ui';
+import { useThemeUI } from 'theme-ui';
+
+const useTheme = (): Theme => {
```

Comment:
Do we need such hook if we can just simply use `const { theme } = useThemeUI();` ?

Reply:
I think we shouldn't resign from this hook because if we use useThemeUI directly, the object value is not typed:
![image](https://user-images.githubusercontent.com/18337813/110241877-7fea3280-7f53-11eb-83d6-2bc294032fd5.png)

We could use it in these ways:
![image](https://user-images.githubusercontent.com/18337813/110241921-c049b080-7f53-11eb-9084-c1095b092da2.png)
or
![image](https://user-images.githubusercontent.com/18337813/110241927-cb044580-7f53-11eb-939b-e1c2748af962.png)

But I think the hook is prettier.

Reply:
Is this hook still working correctly? Maybe you should use the `useTheme` hook from `@emotion/react`?

Reply:
Yes, it's working correctly.